doc/changelog/02-specification-language/14773-revertcast.rst

```diff
@@ -0,0 +1,5 @@
+- **Changed:**
+  Certain cast nodes inserted by tactics were previously printed as normal casts when ``Set Printing All`` was on and otherwise were invisible.
+  They are now printed (and may be input) as ``c :> t`` when ``Set Printing Tactic Casts`` is on.
```

Comment:
```suggestion
  Some cast nodes inserted by tactics were previously printed as normal casts (`` c > t``) when
  ``Set Printing All`` was on and otherwise were invisible.
  They are now printed (and may be entered) as ``c :> t`` when ``Set Printing Tactic Casts`` is on
```

It was truly "some"/"certain" and not "all"?  If you can enter `:>` when `Set Printing Tactic Casts` is not set, then for clarity, make "and may be entered" a separate sentence.